lib/voyager_helpers/liberator.rb

```diff
@@ -586,22 +588,41 @@ def callno_from_852(field_852)
         call_no
       end
 
+      # @return [Fixnum] A ledger ID
+      def get_ledger(conn=nil)
+        query = VoyagerHelpers::Queries.ledger
+        date = Date.today
+        year = date.year
+        month = date.month
+        year += 1 if month > 6
```

Comment:
Is this related to fiscal year?

Reply:
Yes, it's the best way I could think of getting the current fiscal year.

Reply:
Around the end of the fiscal year (June-August), 2 ledgers are open at the same time. So the correct one needs to be determined by some sort of date calculation.